featuretools/primitives/standard/aggregation/max_consecutive_false.py

```diff
@@ -0,0 +1,56 @@
+from woodwork.column_schema import ColumnSchema
+from woodwork.logical_types import Boolean, Integer
+
+from featuretools.primitives.base import AggregationPrimitive
+
+
+class MaxConsecutiveFalse(AggregationPrimitive):
+    """Determines the maximum number of consecutive False values in the input
+
+    Args:
+        skipna (bool): Ignore any `NaN` values in the input. Default is True.
+
+    Examples:
+        >>> max_consecutive_false = MaxConsecutiveFalse()
+        >>> max_consecutive_false([True, False, False, True, True, False])
+        2
+
+        `NaN` values can be ignored with the `skipna` parameter
+
+        >>> max_consecutive_false_skipna = MaxConsecutiveFalse(skipna=False)
+        >>> max_consecutive_false_skipna([False, None, False, False, True, False])
+        2
+    """
+
+    name = "max_consecutive_false"
+    input_types = [ColumnSchema(logical_type=Boolean)]
```

Comment:
Update to work with BooleanNullable? Not sure if things will work with `pd.NA` values present, but we should make sure to test if we add this input type.

Reply:
No, for now let's keep this just as Boolean. The function doesn't work with BooleanNullable (and pd.NA).